src/common/snippets/include/snippets/lowered/pass/brgemm_blocking.hpp

```diff
@@ -112,9 +114,80 @@ class BrgemmBlocking : public snippets::lowered::pass::RangedPass, public Brgemm
             size_t m_block, n_block, k_block;
             std::tie(m_block, n_block, k_block) = get_blocking_params(brgemm_expr);
             modified = mark_blocking_loops(linear_ir, expr_it, m_block, n_block, k_block);
+#ifdef SNIPPETS_DEBUG_CAPS
+            dump_params_to_csv(brgemm_expr, linear_ir);
+#endif  // SNIPPETS_DEBUG_CAPS
         }
         return modified;
     }
+
+#ifdef SNIPPETS_DEBUG_CAPS
+    void dump_params_to_csv(const ov::snippets::lowered::ExpressionPtr& brgemm_expr,
+                            const snippets::lowered::LinearIR& linear_ir) {
+        auto debug_config = linear_ir.get_config().debug_config;
+        const auto brgemm = ov::as_type_ptr<BRGEMM_TYPE>(brgemm_expr->get_node());
+        OPENVINO_ASSERT(brgemm, "Brgemm is nullptr!");
+        auto readEnv = [](const char* envVar) {
+            const char* env = std::getenv(envVar);
+            if (env && *env)
+                return env;
+            return (const char*)nullptr;
+        };
+        const char* dumpParamPath = readEnv("OV_SNIPPETS_DUMP_BRGEMM_PARAMS");
+        if (!dumpParamPath) {
+            return;
+        }
```

Comment:
All the parsing logic should be in `DebugCapsConfig::readProperties()`. Here we need just to read the `debug_config`'s field value